jdaviz/configs/default/plugins/markers/markers.py

```diff
@@ -1,15 +1,17 @@
 import numpy as np
 from astropy import units as u
-from traitlets import Bool, observe
+from astropy.coordinates import SkyCoord
+from traitlets import Bool, Unicode, Instance, observe, default

```

Comment:
I still worry that this is a different pattern than we use everywhere else - can we switch this to import from traitlets instead of having `traitlets.` everywhere?